src/party_menu.c

```diff
@@ -0,0 +1,2723 @@
+        thunk_Sprite_SetPalIndex(partyMenu->sprites[PARTY_MENU_SPRITE_ID_CURSOR], 0);
+        partyMenu->args->context = PARTY_MENU_CONTEXT_0;
+        return PARTY_MENU_STATE_1;
+    } else {
+        partyMenu->args->selectedAction = PARTY_MENU_ACTION_RETURN_10;
+        return PARTY_MENU_STATE_BEGIN_EXIT;
+    }
+}
+
+static int sub_0207C908(PartyMenuStruct *partyMenu) {
+    Pokemon *mon = Party_GetMonByIndex(partyMenu->args->party, partyMenu->partyMonIndex);
+    s32 giratinaResult;
+    int newItem = partyMenu->args->itemId;
+    int oldItem = partyMenu->monsDrawState[partyMenu->partyMonIndex].heldItem;
+    int manipulateItemResult = sub_0207C5D4(partyMenu, mon, &giratinaResult);
+    if (oldItem == ITEM_GRISEOUS_ORB && manipulateItemResult == PARTY_MENU_STATE_PRINT_ITEM_SWAP_MESSAGE && giratinaResult == 0) {
+        manipulateItemResult = PARTY_MENU_STATE_PRINT_GIVE_GRISEOUS_ORB_MESSAGE;
+    }
+    if (oldItem == ITEM_NONE) {
+        ReadMsgDataIntoString(partyMenu->msgData, msg_0300_00107, partyMenu->unformattedStrBuf);
+        BufferBoxMonNickname(partyMenu->msgFormat, 0, Mon_GetBoxMon(mon));
+        BufferItemName(partyMenu->msgFormat, 1, partyMenu->args->itemId);
+        StringExpandPlaceholders(partyMenu->msgFormat, partyMenu->formattedStrBuf, partyMenu->unformattedStrBuf);
+    } else {
+        Bag_AddItem(partyMenu->args->bag, oldItem, 1, HEAP_ID_PARTY_MENU);
+        ReadMsgDataIntoString(partyMenu->msgData, msg_0300_00085, partyMenu->unformattedStrBuf);
+        BufferItemName(partyMenu->msgFormat, 1, oldItem);
+        BufferItemName(partyMenu->msgFormat, 2, newItem);
+        StringExpandPlaceholders(partyMenu->msgFormat, partyMenu->formattedStrBuf, partyMenu->unformattedStrBuf);
+    }
+    DrawFrameAndWindow2(&partyMenu->windows[PARTY_MENU_WINDOW_ID_34], TRUE, 42, 15);
+    FillWindowPixelBuffer(&partyMenu->windows[PARTY_MENU_WINDOW_ID_34], 15);
+    PartyMenu_PrintBufferedMessageOnWindow34(partyMenu);
+    if (partyMenu->args->context == PARTY_MENU_CONTEXT_12) {
+        partyMenu->args->context = PARTY_MENU_CONTEXT_10;
+    }
+    return manipulateItemResult;
+}
+
+static int PartyMenu_HandleChooseMonForInGameTrade(PartyMenuStruct *partyMenu) {
+    if (partyMenu->monsDrawState[partyMenu->partyMonIndex].capsule == 0) {
+        partyMenu->args->selectedAction = PARTY_MENU_ACTION_RETURN_0;
+        return PARTY_MENU_STATE_BEGIN_EXIT;
+    } else {
+        thunk_Sprite_SetPalIndex(partyMenu->sprites[PARTY_MENU_SPRITE_ID_CURSOR], 1);
+        PartyMenu_PrintMessageOnWindow34(partyMenu, msg_0300_00179, TRUE);
+        partyMenu->yesCallback = PartyMenu_ConfirmRemoveCapsuleYes;
+        partyMenu->noCallback = PartyMenu_ConfirmRemoveCapsuleNo;
+        partyMenu->afterTextPrinterState = PARTY_MENU_STATE_YES_NO_INIT;
+        return PARTY_MENU_STATE_WAIT_TEXT_PRINTER;
+    }
+}
+
+u32 sub_0207CA9C(void) {
+    return NARC_plist_gra_plist_gra_00000020_NCGR;
```

Comment:
what graphics are these? can prolly name the functions since you alr dumped them

Reply:
no fucking clue lol